src/classes/STG_Panel.cls

```diff
@@ -1,5 +1,5 @@
 /*
-    Copyright (c) 2014, Salesforce.org
+    Copyright (c) 2021, Salesforce.org
```

Comment:
Older classes do not have to be updated to 2021 - it was only for new classes created in this PR.